logstash/datadog_checks/logstash/logstash.py

```diff
@@ -3,26 +3,35 @@
 from distutils.version import LooseVersion
 
 # 3rd party
-import requests
 from six import iteritems
 from six.moves.urllib.parse import urljoin, urlparse
 
 # project
 from datadog_checks.base import AgentCheck
-from datadog_checks.base.utils.headers import headers
 
 EVENT_TYPE = SOURCE_TYPE_NAME = 'logstash'
 
-LogstashInstanceConfig = namedtuple(
-    'LogstashInstanceConfig', ['service_check_tags', 'tags', 'timeout', 'url', 'ssl_verify', 'ssl_cert', 'ssl_key']
-)
+LogstashInstanceConfig = namedtuple('LogstashInstanceConfig', ['service_check_tags', 'tags', 'url'])
 
 
 class LogstashCheck(AgentCheck):
     DEFAULT_VERSION = '1.0.0'
     DEFAULT_TIMEOUT = 5
     SERVICE_CHECK_CONNECT_NAME = 'logstash.can_connect'
 
+    HTTP_CONFIG_REMAPPER = {
+        'ssl_cert': {
+            'name': 'tls_ca_cert',
```

Comment:
```suggestion
            'name': 'tls_cert',
```
Looks like `ssl_cert` maps to `tls_cert` and not `tls_ca_cert`